sections/model_overview/main.tex

```diff
@@ -1,197 +1,163 @@
-percentage goal.
-
-\begin{equation}
-    \arg \max_{T_i} \quad P(W_i < t)
-\end{equation}
-
-A more sophisticated objective function would be to get the proportion 
-of individuals as close to the percentage aim. 
-In other words, to find the threshold that minimises the difference between the 
-probability and the percentage goal (or maximise its negation).
-
-\begin{equation}\label{eq:obj_queueing_systems}
-    \arg \max_{T_i} \quad -\left( \hat{P} - P(W_i < t) \right)^2
-\end{equation}
+In other words, their aim is to find the threshold that minimises the 
+difference between the probability \(P(W_i < t)\) and the percentage goal 
+(or maximise its negation).
 
-
-The third player, the distributor, has their own choices to make and their own 
+The third player, the ambulance service, has their own choices to make and 
+their own 
 goals to satisfy.
 The strategy set of the third player is the proportion \(0 \leq p_A \leq 1\) of 
-individuals it sends to the first queueing system (the proportion 
-\(p_B = 1 - p_A\) is sent to the second queueing system).
-In addition, the distributor aims to minimise any potential blockages
-that may occur, given the pair of thresholds chosen by the two queueing systems.
-Thus, its objective is to minimise the blocked time of the individuals (\(B_i\))
-that they send to the two queueing systems.
+individuals to send to hospital A.
+Similarly the proportion of individuals to send to hospital B is given by
+\(p_B = 1 - p_A\).
+In addition, the ambulance service aims to minimise any potential blockages
+that may occur, given the pair of thresholds chosen by the two hospitals.
+Thus, its objective is to minimise the blocked time of the individuals 
+(\(B_A\) and \(B_B\))
+that they send to hospitals \(A\) and \(B\).
 Apart from the time being blocked, an additional aspect that may affect the 
-decision of the distributor is the proportion of lost individuals (\(L_i\)).
+decision of the distributor is the proportion of lost individuals 
+\(L_A\) and \(L_B\).
 Equation \ref{eq:obj_distributor} can be used to capture a mixture 
 between the two objectives \(L_i\) and \(B_i\) where \(i \in \{A, B\}\):
-\begin{equation*}\label{eq:obj_distributor}
+\begin{equation}\label{eq:obj_distributor}
     (p_A, p_B) \quad s.t. \quad 
     \alpha L_A(p_A) + (1 - \alpha) B_A(p_A) = 
     \alpha L_B(p_B) + (1 - \alpha) B_B(p_B)
-\end{equation*}
+\end{equation}
 
 Here, \(\alpha\) represents the ``importance'' of each objective,
 where high \(\alpha\) indicates a higher weight on the proportion of lost 
 individuals and smaller \(\alpha\) a higher weight on the time blocked. 
+Note here that the choice of \(p_A\) and \(p_B\) rely solely on equation
+\ref{eq:obj_distributor}.
+An extension that can be applied here is the use of unobservable queues.
+The ambulance service does not necessarily know at which state hospitals A and
+B are at, and thus may need to make decisions based on other
+information~\cite{unobservablequeue}.
```

Comment:
I think this is a bit confused: currently the model **is** an unobservable model: everything is based on rates and expectations.

Reply:
Oh, I think I misinterpreted this issue #25. Will change this accordingly